dotnet/samples/GettingStartedWithProcesses/Step04/Step04_AgentOrchestration.cs

```diff
@@ -21,7 +21,7 @@ namespace Step04;
 public class Step04_AgentOrchestration(ITestOutputHelper output) : BaseTest(output, redirectSystemConsoleOutput: true)
 {
     // Target Open AI Services
-    protected override bool ForceOpenAI => true;
+    //protected override bool ForceOpenAI => true;
```

Comment:
nit: should this be commented out or set to false?

Reply:
Rolled back / re-enabled.